embedded-graphics/src/image/image_bmp.rs

```diff
@@ -26,11 +27,11 @@ use tinybmp::Bmp;
 /// # let mut display: MockDisplay<Rgb565> = MockDisplay::default();
 ///
 /// // Load `patch_16bpp.bmp`, a 16BPP 4x4px image
-/// let image = ImageBmp::new(include_bytes!("../../../assets/patch_16bpp.bmp")).unwrap();
+/// let mut image = ImageBmp::new(include_bytes!("../../../assets/patch_16bpp.bmp")).unwrap();
 ///
 /// // Equivalent behavior
-/// display.draw(&image);
-/// display.draw(image.into_iter());
+/// image.into_iter().draw(&mut display);
+/// image.draw(&mut display);
```

Comment:
This doc comment needs refactoring - the `Note` part doesn't make sense any more, and I think the last two lines could be collapsed to the last `image.draw(&mut display);`. Probably worth checking other doc comments as well. I'll point out any I spot in this review but I might miss some.